src/pages/KadraVykhovnykiv/KVTable.tsx

```diff
@@ -70,6 +71,8 @@ export const KVTable = ({ current, searchData }: props) => {
   }
 
   const fetchData = async () => {
+    if (searchPage != 0)
+    setPage(searchPage);
```

Comment:
1) There are extra spaces
2) Using if without curl braces are not the best idea. (At least add indentation)

Reply:
fixed